GuildedRoseLLC/ItemsViewController.swift

```diff
@@ -28,10 +33,11 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         }
     }
     
-    private func configureDataSource(items: [Item]) {
+    private func configureDataSource(items: [Item] = []) {
         dataSource = ItemCollectionViewDataSource()
```

Comment:
If you wanted, you could initialize this variable in the same place you declare it. That would remove the need for using an optional.